newrelic-agent/src/main/java/com/newrelic/agent/Transaction.java

```diff
@@ -1093,19 +1093,24 @@ private void finishTransaction() {
                             this.getInboundHeaderState().getSyntheticsMonitorId());
                     getIntrinsicAttributes().put(AttributeNames.SYNTHETICS_JOB_ID,
                             this.getInboundHeaderState().getSyntheticsJobId());
-                    getIntrinsicAttributes().put(AttributeNames.SYNTHETICS_TYPE,
-                            this.getInboundHeaderState().getSyntheticsType());
-                    getIntrinsicAttributes().put(AttributeNames.SYNTHETICS_INITIATOR,
-                            this.getInboundHeaderState().getSyntheticsInitiator());
                     getIntrinsicAttributes().put(AttributeNames.SYNTHETICS_VERSION,
-                            this.getInboundHeaderState().getSyntheticsVersion());
-
-                    Map<String, String> attrsMap = this.getInboundHeaderState().getSyntheticsAttrs();
-                    String attrName;
+                            String.valueOf(this.getInboundHeaderState().getSyntheticsVersion()));
```

Comment:
Is the synthetics version supposed to be added to the intrinsic attributes as a `String` or `Number`? And why is it being changed now?

I'm a bit unclear on why there are two different `getSyntheticsVersion()` methods, one that returns an `int` and the other a `String`.

Reply:
According to agent specs, all `synthetics-info` attributes added to the intrinsic attributes should be in the form of a `String`. Re-reading the agent specs, the synthetics version is not _required_ to be added to the intrinsic attributes, so this line can be removed.

There are two `getSyntheticsVersion()` methods, because one version is pulling from the original `synthetics` header and is converting it to an `int` in order to confirm supportability. The second is to retrieve the `String` value of the version that exists within the `synthetics-info` header.